src/components/GlobalLeaderboards/TopAccounts.tsx

```diff
@@ -0,0 +1,154 @@
+import { useState } from "react";
+import { BigNumber } from "ethers";
+import classnames from "classnames";
+import { t } from "@lingui/macro";
+import { formatAmount, formatUsd } from "lib/numbers";
+import { useDebounce } from "lib/useDebounce";
+import Pagination from "components/Pagination/Pagination";
+import TableFilterSearch from "components/TableFilterSearch";
+import Table from "components/Table";
+import { useLeaderboardContext } from "./Context";
+import { USD_DECIMALS } from "lib/legacy";
+import { TableCell, TableHeader } from "components/Table/types";
+import { TopAccountsRow, formatDelta } from "domain/synthetics/leaderboards";
+import AddressView from "components/AddressView";
+import Tooltip from "components/Tooltip/Tooltip";
+
+const parseRow = (s: TopAccountsRow): Record<string, TableCell> => ({
+  id: s.id,
+  rank: s.rank + 1,
+  account: {
+    value: "",
```

Comment:
if i understand correctly `value` won't be used because `render` is passed? maybe better to update types so value is not required? i think passed `value` with empty string might be a bit confusing 

another option: value could be either string or function (i.e. no need to have 2 separate keys)

what do you think?